### PR TITLE
give a better error message for `lxd forkstart`

### PR DESCRIPTION
**Pull request by tych0**
Signed-off-by: Tycho Andersen <tycho.andersen@canonical.com>

### CODE REVIEW

**Comment by stgraber** [comment]
Hmm, can't we do that on the client side instead?

It seems wrong for the server to give instructions to use one particular client.

**Comment by tych0** [comment]
We could by pattern matching on the error message I suppose.

**Comment by stgraber** [comment]
Or just assign another error number.

**Comment by stgraber** [comment]
Though I'm wondering what startup failures we have that shouldn't have the user look at the log.

**Comment by tych0** [comment]
Probably none, I'll move this error message to the client today.

File: lxc/action.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ func (c *actionCmd) run(config *lxd.Config, args []string) error {
 		}
 
 		if err := d.WaitForSuccess(resp.Operation); err != nil {
-			return err
+			return fmt.Errorf("%s\nTry `lxc info --show-log %s` for more info", err, name)
```
Comment (stgraber) [inline lxc/action.go]: That string should be translated.

    fmt.Errorf("%s\n" + gettext.Gettext("Try `lxc info --show-log %s` for more info."), err, name)

Should do it

**Comment by stgraber** [comment]
go fmt :)

**Pull request merged**